src/sentry/logging/formatters.py

```diff
@@ -0,0 +1,29 @@
+"""
+sentry.logging.formatters
+~~~~~~~~~~~~~~~~~~~~~~~~~
+:copyright: (c) 2010-2016 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+
+from __future__ import absolute_import
+
+import traceback
+
+from logging import Formatter
+
+import msgpack
+
+
+class MessagePackFormatter(Formatter):
+    """
+    Return a packed dictionary.
+
+    Will pack the dictionary or the String representation of the message.
+    """
+    def format(self, record):
+        pack = record.msg if isinstance(record.msg, dict) else {'msg': str(record.msg)}
```

Comment:
This `str()` call is dangerous if the value were something non-ascii.

What you'd want is to use `django.utils.encoding:force_bytes` and do:

```python
force_bytes(record.msg, errors='replace')
```

This way you'll never through a `UnicodeDecodeError` here just trying to log something.